### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vaibhavparatwar**
Signed-off-by: Vaibhav Prakash Paratwar <vaibhav.paratwar@seagate.com>

Adding prerequisit, command line to run this test and also hyperlink for refering HARE README.md

-----
[View rendered dtm0/it/all2all/README.md](https://github.com/vaibhavparatwar/cortx-motr/blob/patch-3/dtm0/it/all2all/README.md)

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: dtm0/it/all2all/README.md [hunk 1]
```diff
@@ -23,10 +31,13 @@ What the test does:
    connections from m0d's to the client
  - waits for m0crate completion
 
+## Command to run this test
+sudo ./all2all
```
Comment (sergey-shilov) [inline dtm0/it/all2all/README.md:35]: Please add a code spec around: 
\```sh
sudo ./all2all
\```

File: dtm0/it/all2all/README.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # How to run the test
 
+## Prerequisit
+lnet service should be running check with following command
+```
+sudo systemctl status lnet.service
+```
+
+If not running, please refer troubleshooting section in [README.md](https://github.com/Seagate/cortx-hare/tree/3e97a0ae78340a1b17a84de1f99f84b0b8a349fe#readme)
+
```
Comment (huanghua78) [inline dtm0/it/all2all/README.md:10]: Please use this URL:
```
https://github.com/Seagate/cortx-hare/blob/main/README.md
```

File: dtm0/it/all2all/README.md [hunk 3]
```diff
@@ -23,10 +31,15 @@ What the test does:
...
+```
+
 ## Hare usage
 
 Hare is used for cluster bootstrap and shutdown. Firstly
 it needs to be patched by the hare.patch in current directory
 to make it able to generate clusted configuration that includes
 DTM0 service. After that it should be built and installed into
-the system (see README.md in the root of Hare repo).
+the system (see [README.md](https://github.com/Seagate/cortx-hare/tree/3e97a0ae78340a1b17a84de1f99f84b0b8a349fe#readme) in the root of Hare repo).
```
Comment (huanghua78) [inline dtm0/it/all2all/README.md:45]: https://github.com/Seagate/cortx-hare/blob/main/README.md

File: dtm0/it/all2all/README.md (hunk 2 above)
Comment (vaibhavparatwar) [reply dtm0/it/all2all/README.md:10]: ok

File: dtm0/it/all2all/README.md (hunk 3 above)
Comment (vaibhavparatwar) [reply dtm0/it/all2all/README.md:45]: ok

File: dtm0/it/all2all/README.md [hunk 4]
```diff
@@ -1,5 +1,13 @@
 # How to run the test
 
+## Prerequisit
+lnet service should be running check with following command
+```
+systemctl status lnet.service
```
[the remark this answers is not in the document]
Comment (vaibhavparatwar) [reply dtm0/it/all2all/README.md]: done

File: dtm0/it/all2all/README.md [hunk 5]
```diff
@@ -1,5 +1,13 @@
 # How to run the test
 
+## Prerequisit
+lnet service should be running check with following command
+```
```
[the remark this answers is not in the document]
Comment (vaibhavparatwar) [reply dtm0/it/all2all/README.md:5]: Done

File: dtm0/it/all2all/README.md (hunk 1 above)
Comment (vaibhavparatwar) [reply dtm0/it/all2all/README.md]: Done

**Review by huanghua78 — APPROVED** [review]

**Comment by huanghua78** [comment]
Hello @sergey-shilov , please review this PR again.

**Review by sergey-shilov — APPROVED** [review]
Looks good.

**Pull request merged**